### PR TITLE
Return report description in failure CSV export.

### PR DESCRIPTION
**Pull request by houcros**
Add missing failure description to the CSV export.

### CODE REVIEW

**Comment by houcros** [comment]
With my last commit I had to add support to request the CSV with an extension in the request. The `Accept` header is still supported and preferred. But FE was having a hard time getting it to work like that, it seems the browser is doing some witchcraft with the headers.

File: src/main/scala/com/advancedtelematic/ota/deviceregistry/DevicesResource.scala [hunk 1]
```diff
@@ -223,8 +226,17 @@ class DevicesResource(
           case None      => complete(Errors.InvalidGroupExpression(""))
           case Some(exp) => countDynamicGroupCandidates(ns.namespace, exp)
         } ~
-        (path("failed-installations") & parameter('correlationId.as[CorrelationId])) {
-          cid => fetchFailureStats(cid)
+        (pathPrefix("failed-installations") & parameters('correlationId.as[CorrelationId], 'failureCode.as[String].?)) {
+          (cid, fc) => {
+            rawPathPrefix(".csv") {
+              mapRequest(addAcceptCsvHeader) {
```
Comment (simao) [inline src/main/scala/com/advancedtelematic/ota/deviceregistry/DevicesResource.scala]: you don't need these headers... you might want to return a response with the `Content-Type` header, but not `Accept`, `Accept` is for the client, not the server.

Comment (houcros) [reply src/main/scala/com/advancedtelematic/ota/deviceregistry/DevicesResource.scala]: Mmh but then I'm bypassing the content negotiation, and I thought that was the whole point...
If I can do that then I could return to the first implementation where this endpoint only returns CSV and sets the `Content-Type` header with a single Marshaller?

Comment (simao) [reply src/main/scala/com/advancedtelematic/ota/deviceregistry/DevicesResource.scala]: yes, exactly if you use a .csv ending in the path and return csv in content-type you don't need to do any content negotiation, akka will handle that for you.

**Pull request merged**